api/nodes/serializers.py

```diff
@@ -239,16 +244,52 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
         'subjects',
     ])
 
-    anonymized_fields = [
-        'current_user_can_comment',
-        'forks',
-        'citation',
-        'custom_citation',
-        'node_license',
-        'registrations',
-        'contributors',
-        'bibliographic_contributors',
-        'implicit_contributors',
+    # If you add a field to this serializer, be sure to add to this
+    # list if it doesn't expose user data
+    non_anonymized_fields = [
```

Comment:
Forgot that RegistrationSerializer needs to have non_anonymized fields as well...(this was the main motivator for switching over to anonymized_fields in the first place - these giant lists on these serializers that are subject to change).  Adding now...